command/cli.go

```
@@ -221,6 +216,12 @@ func (cli *CLI) runBinary(configFiles, inspectTasks config.FlagAppendSliceValue,
 	}
 	defer ctrl.Stop()
 
+	// Install the driver after controller has tested Consul connection
```

Good idea to fail fast and install driver later!